clang/lib/Driver/ToolChains/Gnu.h

```diff
@@ -336,6 +338,9 @@ class LLVM_LIBRARY_VISIBILITY Generic_GCC : public ToolChain {
       const llvm::opt::ArgList &DriverArgs,
       llvm::opt::ArgStringList &CC1Args) const override;
 
+  void AddSYCLIncludeArgs(const llvm::opt::ArgList &DriverArgs,
```

Comment:
since this is new function. name it `add...` instead of `Add...`.

Yes, clangDriver naming is very inconsistent, but we try to make them consistent for new functions.